src/app/app.component.ts

```diff
@@ -300,29 +300,31 @@ export class AppComponent implements OnInit {
     initCozy() {
         const root = document.querySelector('[role=application]');
         if (root instanceof HTMLElement) {
-            const data = root.dataset;
+            const data = JSON.parse(root.dataset.cozy);
 
             const protocol = window.location ? window.location.protocol : 'https:';
-            const cozyUrl = `${protocol}//${data.cozyDomain}`;
+            const cozyUrl = `${protocol}//${data.domain}`;
```

Comment:
I think you can remove the selector too and use : `fromDOM` https://github.com/cozy/cozy-client/blob/f8d43b70956006806e2e7b7c276a85ef12bfac43/packages/cozy-client/src/CozyClient.js#L366